libs/ui-watt/src/lib/foundations/typography/storybook/shared/styles.ts

```diff
@@ -0,0 +1,13 @@
+export const typographyHtmlSnippets = {
```

Comment:
Consider renaming this file to snippets, or something else not to confuse with css styles